src/redux/books/books.js

```diff
@@ -1,31 +1,74 @@
+// import axios from 'axios';
+/* eslint-disable camelcase */
+
+// { item_id: 1, title: 'Js', author: 'Nizam khan' },
+// { item_id: 2, title: 'react', author: 'ahmad Khan' },
+// { item_id: 3, title: 'redux', author: 'Mahmood Jan' },
+
```

Comment:
- [ ] Kindly don't commented on old codes. It makes your project not clean. [Requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md#:~:text=Do%20not%20commit%20old%20pieces%20of%20code%20as%20inline%20comments.%20They%20will%20make%20your%20project%20look%20messy.%20If%20you%20need%20to%20review%20a%20previous%20version%20of%20your%20code%2C%20you%20can%20always%20use%20git%20history.)